log4j-core/src/main/resources/META-INF/native-image/org.apache.logging.log4j/log4j-core/resource-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "log4j2.*"
```

Comment:
What is the purpose of this particular include entry? (I get the `META-INF/.../Log4j2Plugins.dat` one.)

Reply:
This entry is for configuration files of the form `log4j2<contextName>.<extension>`. (Unfortunately the `resource-config.json` schema leavers no fields for comments).

Thanks for the comment, that reminded me that `log4j-api` needs a configuration file too for `log4j2.component.properties`, `log4j2.simplelog.properties` and `log4j2.StatusLogger.properties`. Fixed in https://github.com/apache/logging-log4j2/pull/3053/commits/79909eab2699b9f5032c2465a1341f2b175e01aa